### ISSUE TITLE
ByteOutputStream getBytes returns empty Uint8List

### ISSUE THREAD

**Issue by zagronitay** [NicolaVerbeeck/dart_pdf_reader#15]
Hi @NicolaVerbeeck 

ByteOutputStream getBytes returns empty Uint8List
seems like updating position in writeAll func solves this issue.

```
void writeAll(Uint8List current) {
    if (_position + current.length > _bytes.length) {
      _bytes = Uint8List(_bytes.length + current.length + _growSize)
        ..setAll(0, _bytes);
    }
    _bytes.setAll(_position, current);
    _position += current.length;  \\ <--- this line
  }
```

**Comment by NicolaVerbeeck** [comment]
Fix incoming

**Comment by NicolaVerbeeck** [comment]
Fix released in version 0.1.6 on pub.dev

**Comment by zagronitay** [comment]
Thank you!
